genie-agent/src/main/java/com/netflix/genie/agent/cli/RuntimeConfigurationArgumentsImpl.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.agent.cli;
+
+import com.beust.jcommander.Parameter;
+import lombok.Getter;
+
+/**
+ * Implementation of RuntimeConfigurationArguments delegate.
+ *
+ * @author mprimi
+ * @since 4.0.0
+ */
+@Getter
+class RuntimeConfigurationArgumentsImpl implements ArgumentDelegates.RuntimeConfigurationArguments {
+
+    @Parameter(
+        names = {"--launchInJobDirectory"},
+        description = "Whether the job process should be forked within the job directory"
+    )
+    private boolean launchInJobDirectory;
```

Comment:
Should the default be `true` and then have the bdp-cli just always pass in true or vice versa?

Reply:
Default `true` would be a change of default behavior so I don't think it should be.
But this option should be in the default launch argument of the agent launcher, which i forgot.